images/entrypoint.sh

```diff
@@ -62,5 +59,6 @@ cp -f $SRIOV_BIN_FILE $CNI_BIN_DIR
 
 echo "Entering sleep... (success)"
 
-# Sleep forever.
-sleep infinity
+# Sleep forever. 
+# sleep infinity is not available in alpine; instead lets go sleep for ~68 years. Hopefully that's enough sleep
+sleep 2147483647
```

Comment:
I searched and didn't find a simple way other than sleep for a long time. 68 years sounds promising :-)